cli/client.go

```diff
@@ -318,7 +318,7 @@ var clientQueryAskCmd = &cli.Command{
 		}
 
 		fmt.Printf("Ask: %s\n", maddr)
-		fmt.Printf("Price per GigaByte: %s\n", types.FIL(ask.Ask.Price))
+		fmt.Printf("Price per GibiByte: %s\n", types.FIL(ask.Ask.Price))
```

Comment:
I'd probably just make it say
```suggestion
		fmt.Printf("Price per GiB: %s\n", types.FIL(ask.Ask.Price))
```

Reply:
SGTM. Changed!